### PR TITLE
Adding_Installation_ Step_s To_ Readme Fixed #3.

### LINKED ISSUE #3: README - Add Installation Project Steps
Add Installation Project Steps to the README.

Provide the necessary tools in the description.
Specify prerequisites.

### PR DESCRIPTION
**Pull request by alejosuaza1022**
ready

### CODE REVIEW

**Comment by mateoguzmana** [comment]
Thanks for your contribution Alejo! Really glad to see your effort and your motivation on this.

1. Please don't use `_` in the commits and in the title of the PR. To remove this you can just `git commit --amend` and remove the underscores. After that you can re-push your branch with `git push -f origin $branch-name`.

File: README.md [hunk 1]
```diff
@@ -1,2 +1,44 @@
 # FindYourPet
-Find Your Pet web application.
+Find Your Pet web application Install.
+here is the steps to configurate the project in your pc
+- first you must to install git in your pc.
+# GIT INSTALL 
+* **WINDOWS** :
```
Comment (mateoguzmana) [inline README.md]: Don't need to use all in uppercase. Just Windows is okay :)

File: README.md [hunk 2]
```diff
@@ -1,2 +1,44 @@
...
+1. you will to go at this direccion [press here](https://git-scm.com/download/win") here you be able to install it in your pc.
```
Comment (mateoguzmana) [inline README.md]: In all sentences the first letter is uppercase. 

- To download git please go here*** It's more descriptive

File: README.md [hunk 3]
```diff
@@ -1,2 +1,44 @@
...
+2. you will download it.
+3. you must open the executable and follow the steps of installation, when it has finised, you will search in your pc git bash and you will open it.
+
+* **MAC**
```
Comment (mateoguzmana) [inline README.md]: The same for this section

File: README.md [hunk 4]
```diff
@@ -1,2 +1,44 @@
...
+1. you will visit git page in this direccion [press here](https://git-scm.com/download/mac), later you will download git with the steps that you find here.
```
Comment (mateoguzmana) [inline README.md]: you find there***

File: README.md [hunk 5]
```diff
@@ -1,2 +1,44 @@
...
+2. you will open the executable and will finish the installation
+
+* **LINUX**
```
Comment (mateoguzmana) [inline README.md]: Same here

File: README.md [hunk 6]
```diff
@@ -1,2 +1,44 @@
...
+- if you're in a Debian-based distribution like Ubuntu
+*apt-get install git*
+- If you want to install Git on Linux through a binary installer, in general you can do it through the basic package management tool that your distribution brings. If you are in Fedora, you can use yum:
```
Comment (mateoguzmana) [inline README.md]: People doesn't want! They should.

Correct this line with "You should install git on Linux..."

File: README.md [hunk 7]
```diff
@@ -1,2 +1,44 @@
...
+*yum install git-core*
+- if you want more information, or you have another distribution you can vist the [git page](https://git-scm.com/download/linux)
+- later You can verify your version with the next command *git --version*
+# NPM AND NODE.JS INSTALL
```
Comment (mateoguzmana) [inline README.md]: Same here with the uppercases "NPM and Node.js Install"

File: README.md [hunk 8]
```diff
@@ -1,2 +1,44 @@
...
+
+* **WINDOWS**
+you can visit this [page](https://tutobasico.com/instalar-nodejs-y-npm/) for all information about it
```
Comment (mateoguzmana) [inline README.md]: Final point for each sentence.

Comment (mateoguzmana) [reply README.md]: Also, please add links with documentation in English. Remember that tomorrow in our team will be people from the US, Europe, and all around the globe.

File: README.md [hunk 9]
```diff
@@ -1,2 +1,44 @@
...
+* **Linux**
+to steps for install in linux see that [short video](https://www.youtube.com/watch?v=esNlno79dBw)
+
+# INSTALL THE PROJECT
+
+1. open git bash in windows, in mac and linux open the term, for download our project
+* go to the folder where you want that the project is there, and write 
+git clone https://github.com/remic-co/FindYourPet.git
+later to do it, you will have our project in that folder
+* in the term go to the root of the project (file FindYourPet) and run the next commad:
```
Comment (mateoguzmana) [inline README.md]: command*

File: README.md [hunk 10]
```diff
@@ -1,2 +1,44 @@
...
+*npm start*, later you will see in your term the url of our project here will run the project
```
Comment (mateoguzmana) [inline README.md]: If you want to write a command you can use the notation for it with the "`npm start`"

**Review by mateoguzmana — CHANGES_REQUESTED** [review]
Please check the comments below.

**Comment by alejosuaza1022** [comment]
why the command have -r?, what is the deference between  git push origin mu branch and git push -r origin my branch?

**Comment by mateoguzmana** [comment]
I will close this because was duplicate with the another PR. @alejosuaza1022 please do not re-push a different branch, changes in the same branch are ok. :)

**Pull request closed without merging**